src/App.tsx

```diff
@@ -1,61 +1,124 @@
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { User } from './types/User';
+import { FullTodo } from './types/FullTodo';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUserById(userId: number): User | null {
```

Comment:
Consider renaming `getUserById` to `findUserById` to better describe what the function does. 🤔